### PR TITLE
update outdated info

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1712

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @mypil, for your information

Thank you

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 1]
```diff
@@ -34,10 +34,9 @@ Calendar sharing is a global setting, meaning that you, the admin, can enable it
   
 ## Enable calendar sharing using the Microsoft 365 admin center
 
-1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2053743" target="_blank">Services & add-ins</a> page. 
+1. In the admin center, go to the **Settings** \> **Settings**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
1. In the admin center, go to **Settings** \> **Settings**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@kwekuako - This is another doc under @twernermsft, would you be the new author for this?
This PR has been copy edited and is ready for final review.

Thank you.

cc: @kenwith

**Comment by kwekuako** [comment]
Hi Joyce, yes, you can make me the author. I’ve merged the PR. Thanks.

**Pull request merged**